packages/react-native-web-maps/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@teovilla/react-native-web-maps",
-  "version": "0.9.2",
+  "version": "0.9.3",
```

There is no need to update the version manually.
```suggestion
  "version": "0.9.2",
```